src/new/Io/JLD2.jl

```diff
@@ -0,0 +1,18 @@
+
+"""
+    function from_jld2_file(filename::AbstractString,dataset::AbstractString="data")
+
+Loads an object from a JLD2 file given its filename and, optionally, the dataset name.
+"""
+function from_jld2_file(filename::AbstractString,dataset::AbstractString="data")
```

Comment:
Estaria bién añadir una versión de `from_jld2_file` que acepte un tipo en su primer argumento para poder comprovar si el tipo que ha generado `JLD2` es el que el usuario se espera

la firma del mètodo seria:
```julia
from_jld2_file(::Type{T},filename::AbstractString,dataset::AbstractString="data") where T
```

dentro de esta function se puede chequear que el objeto generado `obj`, cumple `isa(obj,T)`

Reply:
Done in 36cc7c3